### DISCUSSION TITLE
Add more customisation options for the gameplay mouse cursor

### DISCUSSION THREAD

**Discussion by DESLiZADORES** [ppy/osu#13191] [Ideas]
**The new feature:** 
The new feature will be included in the future skin editor and will remove the dotted cursor trail. You will be able to change the cursor size, trail, and trail's opacity. You will have to drop 2 already existing cursor and cursortrail files into it. Cursor trails will resemble the ones seen in the alt tab cursor trail bug or cursormiddle.png cursors, but with a more manageable length thats less distracting for players. If players prefer the old trails the feature can be toggled on and off
**Proposal designs of the feature:** 
![image](https://user-images.githubusercontent.com/82096381/113863974-556cde80-9778-11eb-9376-be5ef9632c76.png)
![image](https://user-images.githubusercontent.com/82096381/113864012-6158a080-9778-11eb-9158-355b214a54f8.png)
What I imagine it looking like (if the skin editor will be located in settings)

**Comment by peppy** [comment]
This will be part of a larger skinning system improvement, but it's almost not worth opening issues for things like this until the prerequisites are completed.
